torch/csrc/jit/codegen/cuda/utils.cpp

```diff
@@ -109,6 +110,8 @@ auto parseDebugDumpOptions() {
         options_map[DebugDumpOption::BankConflictInfo] = true;
       } else if (token == "sync_map") {
         options_map[DebugDumpOption::SyncMap] = true;
+      } else if (token == "pass_transform") {
```

Comment:
We may want to use some more specific keyword than "pass_transfrom". Perhaps, "lower_verbose"?

Reply:
done, renamed lower_verbose